Example/Tests/NYTScalingImageViewTests.m

```diff
@@ -9,37 +9,69 @@
 @import UIKit;
 @import XCTest;
 
+#define kGifPath  [[NSBundle bundleForClass:self.class] pathForResource:@"giphy" ofType:@"gif"]
```

Comment:
Can you convert this to a property? We generally avoid `#define`s for this purpose